src/constants.lua

```diff
@@ -68,64 +68,47 @@ constants.DEFAULT_DISTRIBUTION_SETTINGS = {
 	minimumRewardRate = 0.0005, -- 0.05% of the protocol balance after the first year
 	rewardDecayStartEpoch = 365, -- one year of epochs before it kicks in
 	rewardDecayLastEpoch = 547, -- 1.5 years of epochs before it stops (365 + 182)
-	gatewayOperatorRewardRatio = 0.9, -- 90% of the rewards go to the gateway operators
-	observerRewardRatio = 0.1, -- 10% of the rewards go to the observers
+	gatewayOperatorRewardRate = 0.9, -- (90%) the rate of rewards that go to the gateway operators
+	observerRewardRate = 0.1, -- (10%) the rate of rewards that go to the observers
+	missedObservationPenaltyRate = 0.25, -- (25%) penalty for gateways that receive rewards but did not observe
 }
 
--- GAR
+-- Gateway Address Registry
 constants.MIN_WITHDRAWAL_AMOUNT = constants.ARIOToMARIO(1)
 constants.DEFAULT_GAR_SETTINGS = {
 	observers = {
-		maxPerEpoch = 50, -- remove this reference, it is above and defined by the epoch settings
-		tenureWeightDaysDenominator = 180, -- tenure weight is computed by tenure / 6-months
-		tenureWeightPeriod = constants.daysToMs(180), -- 180 days in ms
-		maxTenureWeight = 4, -- i.e. an operator has been running for 2 years or more
+		tenureWeightDurationMs = constants.daysToMs(180), -- 180 days in ms
+		maxTenureWeight = 4, -- the maximum tenure weight, reached when a gateway has been running for 2 years or more
 	},
 	operators = {
 		minStake = constants.ARIOToMARIO(10000), -- 10,000 ARIO
 		withdrawLengthMs = constants.daysToMs(90), -- 90 days to lower operator stake
 		leaveLengthMs = constants.daysToMs(90), -- 90 days that balance will be vaulted
 		failedEpochCountMax = 30, -- number of epochs failed before marked as leaving
-		failedEpochSlashRate = 1, -- 100% of the minimum operator stake is returned to protocol balance, rest is vaulted
-		maxDelegateRewardShareRatio = 95, -- 95% of rewards can be shared with delegates, intended to promote competition between gateways
+		failedGatewaySlashRate = 1, -- (100%) applied to the minimum operator stake, the rest is vaulted
+		maxDelegateRewardShareRatio = 95, -- (95%) the maximum percentage of rewards that can be shared with delegates, intentionally represented as a percentage (vs. rate)
```

Comment:
Double check that this should be a number in the 0-100 range.

Reply:
Name is ratio but might actually be percentage?

Reply:
it is a percentage - but that would require us backfilling the `minDelegateRewardShareRatio` on gateway settings so to keep it as is for now